pkg/controller/job/job_controller.go

```diff
@@ -280,6 +280,11 @@ func newDaemonSetForCR(cr *storageosv1alpha1.Job) (*appsv1.DaemonSet, error) {
 		},
 	}
 
+	// Add pod tolerations if defined.
+	if len(cr.Spec.Tolerations) > 0 {
+		dset.Spec.Template.Spec.Tolerations = cr.Spec.Tolerations
```

Comment:
We need the toleration exists and value check here as well.

Reply:
You're totally right, my bad. Sorry for so much inconvenience.